app/vote_ext_utils.go

```diff
@@ -498,8 +489,10 @@ func checkNonRpVoteExtensionsSignatures(ctx sdk.Context, extVoteInfo []abciTypes
 
 		_, validator := validatorSet.GetByAddress(valAddr)
 		if validator == nil {
+			if ctx.BlockHeight() < helper.GetDisableValSetCheckHeight() || ctx.BlockHeight() >= helper.GetTallyFixHeight() {
```

Comment:
I think we can move this check in separate function and use it from all places and I think it should be `ctx.BlockHeight() >= helper.GetDisableValSetCheckHeight() && ctx.BlockHeight() < helper.GetTallyFixHeight()`

Reply:
Yea, your right.